eras/conway/impl/src/Cardano/Ledger/Conway/TxInfo.hs

```diff
@@ -205,9 +205,6 @@ instance Crypto c => EraPlutusTxInfo 'PlutusV1 (ConwayEra c) where
   toPlutusScriptPurpose proxy = transPlutusPurposeV1V2 proxy . hoistPlutusPurpose toAsItem
 
   toPlutusTxInfo proxy pp epochInfo systemStart utxo tx = do
-    let refInputs = txBody ^. referenceInputsTxBodyL
-    unless (Set.null refInputs) $ Left $ inject $ ReferenceInputsNotSupported @(ConwayEra c) refInputs
-
     timeRange <- Alonzo.transValidityInterval pp epochInfo systemStart (txBody ^. vldtTxBodyL)
     inputs <- mapM (Babbage.transTxInInfoV1 utxo) (Set.toList (txBody ^. inputsTxBodyL))
```

Comment:
`Babbage.transTxInInfoV1` will fail for reference scritps. There is more fixing and testing that needs to be done. New functions like these need to be added to Conway.TxInfo

https://github.com/IntersectMBO/cardano-ledger/blob/60e0dcc2da3824e48c46066d917b0eee08947699/eras/babbage/impl/src/Cardano/Ledger/Babbage/TxInfo.hs#L149-L152

https://github.com/IntersectMBO/cardano-ledger/blob/60e0dcc2da3824e48c46066d917b0eee08947699/eras/babbage/impl/src/Cardano/Ledger/Babbage/TxInfo.hs#L101-L102